cloudman/helmsman/tests/data/helmsman_config.yaml

```diff
@@ -7,9 +7,11 @@ charts:
   cvmfs:
     name: cloudve/galaxy-cvmfs-csi
     namespace: cvmfs
+    create_namespace: true
```

Comment:
Do we need this? I feel like the namespace should be implicitly created?

Reply:
I thought it shouldn't be implicit for helmsman for scenarios (like Anvil) where we want charts to be strictly deployed within the boundaries of a namespace?

Reply:
In that scenario, won't we be passing the correct namespace in anyway, only it will already exist? I feel like we'll have a redundant create_namespace=true everywhere.

Reply:
I was hoping we wouldn't have it anywhere. It's not even needed for CVMFS because `cloudman-boot` is adding it, I just added it here for the test. I added the functionality in case it will be needed, but I don't see any scenario where we need the namespace to be created by helmsman behind the scenes

Reply:
So as discussed, this is needed and we should leave it in after all.